test/jtreg-ext/requires/VMProps.java

```diff
@@ -602,9 +602,10 @@ private void printLogfileContent(Map<String, String> logFileNames) {
 
     private boolean checkDockerSupport() throws IOException, InterruptedException {
         log("checkDockerSupport(): entering");
-        ProcessBuilder pb = new ProcessBuilder(Container.ENGINE_COMMAND, "ps");
-        Map<String, String> logFileNames = redirectOutputToLogFile("checkDockerSupport(): <container> ps",
-                                                      pb, "container-ps");
+        ProcessBuilder pb = new ProcessBuilder("which", Container.ENGINE_COMMAND);
+        Map<String, String> logFileNames =
+            redirectOutputToLogFile("checkDockerSupport(): which <container-engine>",
```

Comment:
Maybe it's better for the message to include the actual name of the engine?

```
redirectOutputToLogFile("checkDockerSupport(): which " + Container.ENGINE_COMMAND),
```

Reply:
I don't see how this comment was resolved. Did I miss something?

Reply:
Sorry I missed this one. Filed: [JDK-8318029](https://bugs.openjdk.org/browse/JDK-8318029)
Minor improvement to logging output in container at-requires
Will address soon.

Reply:
Thanks for the follow-up bug. I linked [JDK-8318029](https://bugs.openjdk.org/browse/JDK-8318029) to the original.